testjson/execution.go

```diff
@@ -131,6 +131,23 @@ func (p *Package) TestCases() []TestCase {
 	return tc
 }
 
+// EarliestTime returns the earliest start time found within a child test case,
+// and a bool for if any such time was found.
+func (p *Package) EarliestTime() (time.Time, bool) {
+	found := false
+	t := time.Time{}
+
+	for _, tc := range p.TestCases() {
+		if !found || tc.Time.Before(t) {
+			t = tc.Time
+		}
+
+		found = true
+	}
+
+	return t, found
```

Comment:
I think we don't need the bool return value here. Checking if the returned value `IsZero` should be fine.

Reply:
I actually originally did that, but thought it was more likely to get flagged in the PR 😅 Sounds good I will change both of these 👍